java/metrics/src/test/scala/org/typelevel/otel4s/java/metrics/ObservableSuite.scala

```diff
@@ -62,14 +62,15 @@ class ObservableSuite extends CatsEffectSuite {
         .use(_ =>
           sdk.metrics
             .map(
-              _.flatMap(_.data.points.map(x => (x.value, x.attributes)))
-            )
-            .assertEquals(
-              List(
-                (1336.0, List(Attribute("1", "2"))),
-                (1337.0, List(Attribute("a", "b")))
-              )
+              _.flatMap(
+                _.data.points.map(x => (x.value, x.attributes))
+              ) sameElements
```

Comment:
I think [`sameElements`](https://www.scala-lang.org/api/2.13.3/scala/collection/immutable/List.html#sameElements[B%3E:A](that:scala.collection.IterableOnce[B]):Boolean) is going to have the same issue?

> Are the elements of this collection the same (and in the same order) as those of that? 

This compiles, but is awkward:

```scala
          sdk.metrics
            .map(
              _.flatMap(
                _.data.points.map(x => (x.value, x.attributes))
              ).toSet
            )
            .assertEquals(
              Set[(Any, List[Attribute[_]])](
                (1336.0, List(Attribute("1", "2"))),
                (1337.0, List(Attribute("a", "b")))
              )
            )
```

Reply:
Yes you are right :crying_cat_face: